README.md

```diff
@@ -8,6 +8,8 @@ AWS related modules and plugins supported by the Ansible community are in the [c
 
 This collection has been tested against following Ansible versions: **>=2.9.10**.
 
```

Comment:
I guess we don't need to bring any change to README.md file.

Reply:
hmmm, this happen in the backport-2 release for 2.4.0 also https://github.com/ansible-collections/community.aws/pull/1035

Reply:
This PR is supposed to only update the changelog in the main branch with what we released in 2.2.0. Not fully understand why we need to update the README.md with stable-2.